### ISSUE TITLE
Uploading MTG Arena Collection Doesn't Seem To Work

### ISSUE THREAD

**Issue by brazberry** [HearthSim/LotusTracker#118]
I opened LotusTracker and then MTG Arena. Then I navigated to the collection page. I see some logs in the preferences tab that says its loading x collection with x cards. However when I go to the collection tab https://www.mtglotusvalley.com/user I don't see any cards.

**Comment by edipo2s** [comment]
Hi, have you success logged in on Lotus Tracker? You can check that doing a right click on Lotus Tracker tray icon and check if there's an **My Profile** and **Logout** options.

**Comment by brazberry** [comment]
Yes I see a My Profile and Logout option when I right click on the tray. I also can see all my matches on the website when I click on the My Profile link. However when I click on the collection tab on that page or when I click on my user name in the top right corner and then click on My Collection (These seem to go to different screens for some reason), I don't see any of my cards.
